components-api/src/main/scala/pl/touk/nussknacker/engine/api/definition/ParameterValidator.scala

```diff
@@ -256,6 +255,31 @@ case object JsonValidator extends ParameterValidator {
 
 }
 
+case class ValidationExpressionParameterValidatorToCompile(
```

Comment:
Can you move `ValidationExpressionParameterValidator` into the `interpreter` module? It is important because:
1. `CompiledExpression` leaked to the `component-api` and I want it to came back into `extensions-api`
2. In the future I would like to split `Parameter` into two classes: accessible for the component developers and second one, ready to use by compiler, with defined properties - see TODO there. This change will help to achieve that

I know that there will be a little issue with the fact that `ParameterValidator` is sealed trait and we use it with circe, but I belive that you will handle it.

Reply:
do you mean for me to also move `CompileExpression` to `extensions-api`? from what i quickly checked, it doesn't seem to be a simple move, and it shouldn't be a part of this PR imo

Reply:
You are right, I forgot that it accidentally leaded also into here: https://github.com/TouK/nussknacker/pull/5373/files#diff-dcbf8a2291f73522ea043e9e7d33ddb0dc8317c3c513c90b9c9a7a13a61e7e7f 
I already have a PR that fixes it. I'll move this class into the extension-api after merge of this PR